pootle/apps/pootle_app/views/index/index.py

```diff
@@ -28,7 +28,9 @@ def view(request):
     lang = request.COOKIES.get(COOKIE_NAME, None)
 
     if lang is None:
-        supported = Language.live.cached_dict()
+        supported = Language.live.cached_dict(
```

Comment:
Yes, it is untested. But it was untested before too. This code is just touched to pass the param. I am not adding tests for this in this PR.